cluster/gce/trusty/configure.sh

```diff
@@ -45,8 +45,10 @@ config_ip_firewall() {
 create_dirs() {
   # Create required directories.
   mkdir -p /var/lib/kubelet
-  mkdir -p /var/lib/kube-proxy
   mkdir -p /etc/kubernetes/manifests
+  if [ "${KUBERNETES_MASTER:-}" = "false" ]; then
```

Comment:
Style guide says these should be [[

https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B